src/App.jsx

```diff
@@ -1,24 +1,48 @@
 import NavBar from './components/NavBar';
-import { BrowserRouter as Router, Routes, Route} from 'react-router-dom';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
```

Comment:
We're not using React Router, so it can be removed. You can remove the import here but also in the package.json, you can use the command "npm uninstall react-router-dom" for that

Reply:
This breaks if I don't use the router, so I put it on my list to look into.